package.json

```diff
@@ -21,5 +21,8 @@
     "vite-tsconfig-paths": "^4.3.2",
     "vitest": "^2.1.3"
   },
-  "packageManager": "pnpm@8.15.5"
+  "packageManager": "pnpm@8.15.5",
+  "resolutions": {
+    "zod": "3.22.4"
+  }
```

Comment:
Hm I believe we've moved away from doing this (@manugoyal @ibolmo have context)

Reply:
In this PR's case, the version of zod was different between the proxy and SDK, which was causing some strange type errors. I guess we could manually ensure they are the same?

Reply:
I think the way we accomplished this in the past was by pinning it in `pnpm-lock.yaml` instead. @manugoyal or @ibolmo can help you do this the right way.

Reply:
Will defer to @ibolmo, as I was only familiar with the `resolutions` approach, but now see we are using overrides in other parts of the codebase, such as our internal repo. A quick google suggests that `pnpm.overrides` is preferred for pnpm projects.

Reply:
Yeah I did an `override`: https://github.com/braintrustdata/braintrust-sdk/pull/469/files